src/JoomlaBrowser.php

```diff
@@ -61,18 +95,18 @@ public function doAdministratorLogin($user = null, $password = null)
 		}
 
 		$this->debug('I open Joomla Administrator Login Page');
-		$I->amOnPage('/administrator/index.php');
-		$I->waitForElement(['id' => 'mod-login-username'], 60);
+		$I->amOnPage($locator::$adminLoginPageUrl);
+		$I->waitForElement($locator::$adminLoginUserName, 60);
```

Comment:
We started in gsoc project with it - can you define the timeout as constant. e.g. const TIMEOUT = 60;

and than use it here?